examples/ffi/crashtracking.c

```diff
@@ -41,7 +49,6 @@ int main(int argc, char **argv) {
   }
 
   // raise(SIGSEGV);
```

Comment:
Does this still need to be here?  If it's useful, might be interesting to toggle with a preprocessor macro.

Reply:
done